src/test/acceptance/features/are-you-pregnant.feature

```diff
@@ -12,15 +12,44 @@ Feature:
   Scenario: Yes and No are displayed
     Then Yes and No options are displayed
 
-  Scenario: Select the no option
-    When I select the no option
+  Scenario: Do not select an option
+    When I do not select an option
+    And I click continue
+    Then I am informed that I need to select an option
+
+  Scenario: Select the No option
+    When I select the No option
+    And I click continue
     Then I am shown the card address page
 
-  Scenario: Select the yes option and enter a valid expected due date
-    When I select the yes option
-    And I enter my expected due date in six months time
+  Scenario: Select Yes and the expected date of delivery instructional text is shown
+    When I select the Yes option
+    Then expected date of delivery instructional text is displayed
+
+  Scenario: Select the Yes option and expected date of delivery appears
+    When I select the Yes option
+    And I enter a valid expected delivery date
+    And I click continue
     Then I am shown the card address page
 
-  Scenario: Do not select an option
-    When I do not select an option
-    Then I am informed that I need to select an option
+  Scenario: Select the Yes option and do not enter a date
+    When I select the Yes option
+    And I click continue
+    Then I am informed that I need to enter an expected delivery date
+
+  Scenario: Select the Yes option and enter text for the delivery date
+    When I select the Yes option
+    And I enter text in the expected delivery date fields
+    Then no values are present in the expected delivery date fields
+
+  Scenario: Select the Yes option and enter a date too far in the past
```

Comment:
I would reword this to say what is too far in the past, i.e. more than one month. Same comment for the too far in the future scenario.